src/tests/test_configobj.py

```diff
@@ -1375,3 +1375,28 @@ def test_writing_out_dict_value_with_unrepr(self):
         c = ConfigObj(cfg, unrepr=True)
         assert repr(c) == "ConfigObj({'thing': {'a': 1}})"
         assert c.write() == ["thing = {'a': 1}"]
+
+
+def test_dividers_happy(cfg_contents):
+    cfg = cfg_contents("""
+[section]
+    ook : 'eek'
+    monkey = True
+    beast + 666
+    """)
+    c = ConfigObj(cfg, dividers=':=+')
+    assert c['section'] is not None
+    assert c['section']['ook'] == 'eek'
+    assert c['section']['monkey'] == 'True'
+    assert c['section']['beast'] == '666'
+
+
+def test_dividers_unhappy(cfg_contents):
+    cfg = cfg_contents(u"""
+[section]
+    ook … 'eek'
+    """)
+    try:
+        c = ConfigObj(cfg, dividers=u'…')
+    except AttributeError as err:
+        assert str(err) == "No valide characters found for dividers."
```

Comment:
Great that you added unit tests!

There's one more test case I'd like to see, which is testing what happens when more than one divider appears on a line.  For example, I'd expect if someone set ``dividers=':='`` and then said

```
equation: p=np
```

then either ``equation`` will end up as ``'p=np'`` or the line will be treated as having more than one divider and will result in an error.  Either way we should have a test for this, since it's a new edge case.  (We should probably add a note in the docs you added explaining this as well.)